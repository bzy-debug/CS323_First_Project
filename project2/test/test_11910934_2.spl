int test_11910934_2()
{
  int x1 = 5;
  6 = x1;
  x1 = t +3;
  return 0;
}
