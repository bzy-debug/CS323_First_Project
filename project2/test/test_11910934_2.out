Error type 6 at Line 8: left side in assignment is rvalue
Error type 1 at Line 5: undefined variable: t
Error type 7 at Line 10: unmatching operands
Error type 5 at Line 10: unmatching type on both sides of assignment
